<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatWidget</class>
 <widget class="QWidget" name="ChatWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTextEdit" name="chatText">
     <property name="autoFormatting">
      <set>QTextEdit::AutoBulletList</set>
     </property>
     <property name="undoRedoEnabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="messageText"/>
     </item>
     <item>
      <widget class="QPushButton" name="sendButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Send</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>messageText</tabstop>
  <tabstop>sendButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>messageText</sender>
   <signal>textChanged(QString)</signal>
   <receiver>ChatWidget</receiver>
   <slot>messageTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>31</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>1</x>
     <y>258</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sendButton</sender>
   <signal>clicked()</signal>
   <receiver>ChatWidget</receiver>
   <slot>emitSendMessage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>297</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>messageText</sender>
   <signal>returnPressed()</signal>
   <receiver>ChatWidget</receiver>
   <slot>emitSendMessage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>110</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>sendMessage(QString)</signal>
  <slot>emitSendMessage()</slot>
  <slot>messageTextChanged(QString)</slot>
 </slots>
</ui>
